db/version_edit.h

```diff
@@ -241,6 +241,57 @@ struct LevelFilesBrief {
   }
 };
 
+using WalNumber = uint64_t;
+
+// Information of a new WAL tracked by VersionEdit.
+class AddedWal {
+ public:
+  AddedWal() = default;
```

Comment:
Note that this leaves `number_` and `bytes_` essentially uninitialized; consider using in-class initializers. (Affects `DeletedWal` too.)